mtv-migration/mtv.sw.yaml

```diff
@@ -69,12 +75,36 @@ states:
           toStateData: .migrationErrorMessage
         functionRef:
           refName: getMigrationError
+      - name: createNotification
+        functionRef:
+          refName: createNotification
+          arguments:
+            recipients:
+              type: "entity"
+              entityRef: .recipients
+            payload:
+              title: "MTV migration failed"
+              description: '"MTV migration " + .migrationName + " failed. Error: " + .migrationErrorMessage'
+              scope: "SonataFlow"
+              topic: "MTV migration workflow"
     stateDataFilter:
       output: '{migrationError: .migrationErrorMessage}'
     end: true
   - name: MigrationSuccessful
     type: operation
-    actions: []
+    actions:
+      - name: createNotification
+        functionRef:
+          refName: createNotification
+          arguments:
+            recipients:
+              type: "entity"
+              entityRef: .recipients
+            payload:
+              title: "MTV migration successful"
+              description: '"MTV migration " + .migrationName + " succeeded"'
```

Comment:
how difficult would it be to add a link to a virtual machine?

Reply:
we actually did something like that for create-vm workflow:
https://github.com/parodos-dev/serverless-workflows/blob/main/request-vm-cnv/request-vm-cnv.sw.yaml#L243

Reply:
it is a list of VMs. not a single VM.

Reply:
is there other way to reference the outcome of the migration?

Reply:
not that I know of. I can try adding a link to the plan. not certain I have access to the OCP URL